tests/test_docs.py

```diff
@@ -0,0 +1,102 @@
+# Copyright 2022 MosaicML Composer authors
```

Comment:
Any reason not to delete the `.ci/test_docs.py` file?

Reply:
it's used in Jenkins. We won't nuke it until we're fully migrated off

Reply:
oh its still used for nightlies. fair enough